### PR TITLE
Update root.rb

### PR DESCRIPTION
**Pull request by cxwx**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by cxwx** [comment]
root add fits feature

File: Formula/root.rb [hunk 1]
```diff
@@ -5,13 +5,13 @@ class Root < Formula
...
-    sha256 "acbac57657964414945706d499bab86ad7ac6367c78493a5e47de2eba111e0f1" => :high_sierra
-    sha256 "82e6f8dc5048f3977c9a942c2fd62671423f95bfe77e6d5abd9f53b6ac141b13" => :sierra
-  end
+  
+ # bottle do
+ #   sha256 "83090dd063a5cde66bf587f6000b798fda756476fee3651b6f660e1b39cff4bb" => :catalina
+ #   sha256 "9ce760ff961b29b382d8373c6fbe72808e025b9a3e6113deb64df8586b0a853d" => :mojave
+ #   sha256 "acbac57657964414945706d499bab86ad7ac6367c78493a5e47de2eba111e0f1" => :high_sierra
+ #   sha256 "82e6f8dc5048f3977c9a942c2fd62671423f95bfe77e6d5abd9f53b6ac141b13" => :sierra
+ # end
```
Comment (Moisan) [inline Formula/root.rb]: Please do not change the `bottle` block. It is automatically updated when we merge the pull request.

**Review by Moisan — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
poke @henryiii who is our “master of ROOT”

Given that Homebrew has a `cfitsio` formula, could we rely on that instead of building one inside root?

**Comment by henryiii** [comment]
We should be able to use the external package; in conda-forge we build with an external cfitsio:

https://github.com/conda-forge/root-feedstock/blob/master/recipe/meta.yaml#L82

**Comment by henryiii** [comment]
Add:

```ruby
depends_on "cfitsio"
```

And revert the built-in line. I *think* root will automatically enable it if it finds it, it has lots of optional features it treats this way, but go ahead and be explicit and keep the cfitsio=ON option.

**Pull request closed without merging**